shared/src/types/Session.ts

```diff
@@ -7,23 +7,18 @@ export enum SessionType {
   private = 'private',
 }
 
-// Input to DB
-export type ExerciseStateInput = Omit<ExerciseState, 'timestamp'> & {
+// Data stored in DB
+export type ExerciseStateData = Omit<ExerciseState, 'timestamp'> & {
   timestamp: Timestamp;
 };
 
-export type SessionInput = Omit<Session, 'exerciseState' | 'startTime'> & {
-  exerciseState: ExerciseStateInput;
+export type SessionData = Omit<Session, 'exerciseState' | 'startTime'> & {
```

Comment:
we should probably omit the `hostProfile` field here as well 🤔

Reply:
Ah ok, probably only possible to get the host profile through firebase-admin auth and not the client's...
hm... cause without omitting this prop here we would get inaccurate `Session` types on the client subscriptions (and probably other places) since it actually does not contain `hostProfile`.

Reply:
Bummer, don't think we can get it from auth on the client, and that is a good thing, but it messes things up here. Maybe we should actually store the hostProfile on the session document then?

Reply:
But if we store it it can get out of sync. Such a pity to have to make a request for that

Reply:
Updated the types and created a separate type ”with host profile" [8426d5c](https://github.com/29ki/29k/pull/1004/commits/8426d5c7e5d3eb6cb69ef92bd135086371bcb232)